### PR TITLE
Updating the buildtools version to the latest

### PR DESCRIPTION
**Pull request by Anipik**
This PR doesnot change any functionality of any sought.
During the netcoreapp3.0 build on ubuntu, few  restore errors are printed.

```
/git/machinelearning/Tools/crossgen/crossgen.csproj : error NU1202: Package Microsoft.NETCore.App 3.0.0-preview-27324-5 is not compatible with netcoreapp2.0 (.NETCoreApp,Version=v2.0). Package Microsoft.NETCore.App 3.0.0-preview-27324-5 supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0)
/git/machinelearning/Tools/crossgen/crossgen.csproj : error NU1202: Package Microsoft.NETCore.App 3.0.0-preview-27324-5 is not compatible with netcoreapp2.0 (.NETCoreApp,Version=v2.0) / linux-x64. Package Microsoft.NETCore.App 3.0.0-preview-27324-5 supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0)
```

These errors doesnot interfere with the build or tests. This PR resolves those errors as well.

### CODE REVIEW

**Comment by danmosemsft** [comment]
Please share some context for reviewers. Your goal here is to ingest your https://github.com/dotnet/buildtools/pull/2229. Does this change here modify the target framework for any tests, or leave the status quo, what do you need it for, etc.

**Comment by danmosemsft** [comment]
Better still you could link the issue. 😃

File: init-tools.sh [hunk 1]
```diff
@@ -6,12 +6,14 @@ __PACKAGES_DIR="$__scriptpath/packages"
 __TOOLRUNTIME_DIR="$__scriptpath/Tools"
 __DOTNET_PATH="$__TOOLRUNTIME_DIR/dotnetcli"
 __DOTNET_CMD="$__DOTNET_PATH/dotnet"
+__TARGET_FRAMEWORK="netcoreapp2.1"
 if [ -z "${__BUILDTOOLS_SOURCE:-}" ]; then __BUILDTOOLS_SOURCE=https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json; fi
 export __BUILDTOOLS_USE_CSPROJ=true
 __BUILD_TOOLS_PACKAGE_VERSION=$(cat "$__scriptpath/BuildToolsVersion.txt" | sed 's/\r$//') # remove CR if mounted repo on Windows drive
 
 if [[ $2 == *"Intrinsics"* ]]; then
```
Comment (safern) [inline init-tools.sh]: Is it guaranteed to always be  passed as a second argument?

Comment (Anipik) [reply init-tools.sh]: yes you are right, it needs to be changed to loop through all the arguments

**Comment by Anipik** [comment]
@eerhardt @safern can you take a look at this ?

**Pull request merged**